client/src/components/History/Content/Dataset/DatasetDetails.vue

```diff
@@ -22,21 +22,52 @@
                     </div>
                 </div>
                 <DatasetActions :item="result" />
+                <!-- Peek is safe content for v-html -->
+                <!-- eslint-disable-next-line vue/no-v-html -->
                 <pre v-if="result.peek" class="dataset-peek p-1" v-html="result.peek" />
             </div>
         </div>
+        <div v-else class="dataset">
+            <!--
+                placeholder skeleton for loading smoothly without a spinner
+                (which you don't even see since this happens pretty fast)
+            -->
+            <div class="p-2 details">
+                <div class="summary">
+                    <div class="mb-1">
+                        <b-skeleton width="85%"></b-skeleton>
```

Comment:
This is neat but in this case it leads to flickering for me because its too fast.

Reply:
The placeholder goes away too fast, or the animation moves too fast?

The problem I'm trying to work around here is that we want the animation to start as soon as the click happens for a satisfying interaction there.  We fire the details request and need a placeholder of some sort approximating the space that will be used by the resultant details to animate against, but we can't know that perfectly.